zk/txpool/pool.go

```diff
@@ -2513,11 +2512,11 @@ type WorstQueue struct {
 	pendingBaseFee uint64
 }
 
-func (p WorstQueue) Len() int { return len(p.ms) }
-func (p WorstQueue) Less(i, j int) bool {
+func (p *WorstQueue) Len() int { return len(p.ms) }
```

Comment:
Although this may boost perf, I'm not 100% sure we can do it. @hexoscott interested in your thoughts.